adds citation and manafest files

alan-stokes opened the pull request:
adds citation and manifest files. start point for further work on creating a doi for the tools

alan-stokes committed:
adds citation and manafest files

alan-stokes committed:
added either _version.py or added __version_day__ to the _version file

alan-stokes committed:
fixing from donals feedback

alan-stokes committed:
added pypitoimport map

alan-stokes committed:
removed data-released (as generated by other script) and added aggregator script for python dependancies

dkfellows commented on nengo_spinnaker_gfe/machine_vertices/interposer_machine_vertex.py:
```diff
@@ -84,7 +84,10 @@ def generate_machine_data_specification(self, spec, placement,
                                             reverse_iptags,
                                             machine_time_step,
                                             time_scale_factor):
-        pass
```
The changes to this file do not belong on this branch.

alan-stokes replied:
wonder how they got in. ah well. commited to removal now.

alan-stokes committed:
removal of random code

alan-stokes committed:
adds the flag for identifier for the dois and added the intergration into FEC

alan-stokes committed:
https://www.youtube.com/watch?v=YHaNo54iTHo

alan-stokes committed:
adds the scripts to spinnutils instead of support scripts and adds the funcitonality for bibtexing the citation file.

needs testing the codes again, esp the doi stuff

alan-stokes committed:
ready to test with version updator

alan-stokes committed:
little fixes.

posting the formatted from yalm dump for citation file,s as they likely more better.

releaser has some mods to make it easiewr to debug, and a ton of little fixes

moved file compression to before any requests. on hope the delay was causing crash. alas not. but not worth reverting

alan-stokes committed:
removed pypi requirements which have same name. as requested by donal

alan-stokes committed:
fixes issue with manifest and adds the extras needed for the aggreator to work
